README.md

```diff
@@ -1,67 +1,58 @@
 # Stylix
 
-Stylix is a NixOS module which applies the same colour scheme, font and
-wallpaper to a range of applications and desktop environments.
+## About
 
-## What's this?
+Stylix is a theming framework for [NixOS](https://nixos.org), [Home
+Manager](https://github.com/nix-community/home-manager), and
+[nix-darwin](https://github.com/LnL7/nix-darwin) that applies color schemes,
+wallpapers, and fonts to a wide range of applications.
 
-[base16.nix](https://github.com/SenchoPens/base16.nix#readme) allows you to
-import colours from [base16](https://github.com/chriskempson/base16#readme)
-into Nix code. Stylix takes this a step further:
-
-- Automatically colours and changes the font of apps
-- Sets your wallpaper
-- Exports the colour scheme to be used manually for anything we missed
-- Can also generate themes based on an image
-
-For those not familiar with [NixOS](https://nixos.org/) and
-[Home Manager](https://github.com/nix-community/home-manager#readme):
-
-- NixOS is a Linux distribution
-- Home Manager is a program which runs anywhere
-- Both use the Nix language and package manager
-- Both let you install programs and change settings via code
-
-Stylix supports either NixOS + Home Manager, or Home Manager on its own.
-Certain features are only available with NixOS.
+Unlike color scheme providers such as
```

Comment:
To me this implies the linked repositories are providing the color schemes themselves (as in [`tinted-theming/schemes`](https://github.com/tinted-theming/schemes)) rather than a way to use them from Nix.

```suggestion
Unlike color scheme utilities such as
```